packages/devtools_app/lib/src/shared/globals.dart

```diff
@@ -15,6 +15,13 @@ import 'framework_controller.dart';
 import 'preferences.dart';
 import 'survey.dart';
 
+/// Whether this DevTools build is external.
+bool get isExternalBuild => _isExternalBuild;
+bool _isExternalBuild = true;
+
+/// Flag the build as external.
+void setExternalBuild() => _isExternalBuild = true;
```

Comment:
my comment still stands about making this `late final isExternalBuild`. We never want this value changed after the initial assignment.

Reply:
The problem is that test environment never initializes this flag and thus fails with non initialized late field.
We do not have a shared test configuration point, right?

Reply:
I can add flag like 'bool _sealed' and set it in setExternalBuild, though. How does it sound?